charts/spire/values.yaml

```diff
@@ -101,3 +98,15 @@ keystore:
 
 torProxy:
   enabled: true
+
+# Environment variable listing
+env: {}
+  # non sensitive variables
+  # refer to https://github.com/chronicleprotocol/oracle-suite/tree/master/cmd/gofer#environment-variables
+  # normal:
+  #   CFG_WEBAPI_LISTEN_ADDR: "0.0.0.0:8080"
+
+# This is a config file for gofer. it can override the default config if supplied
```

Comment:
```suggestion
  # refer to https://github.com/chronicleprotocol/oracle-suite/tree/master/cmd/spire#environment-variables
  # normal:
  #   CFG_WEBAPI_LISTEN_ADDR: "0.0.0.0:8080"

# This is a config file for Spire. it can override the default config if supplied
```